app/src/main/res/layout/recent_patients.xml

```diff
@@ -6,22 +6,25 @@
   tools:context="org.simple.clinic.recentpatientsview.RecentPatientsView"
   tools:parentTag="org.simple.clinic.recentpatientsview.RecentPatientsView">
 
-  <androidx.recyclerview.widget.RecyclerView
-    android:id="@+id/recentRecyclerView"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:clipToPadding="false"
-    android:paddingBottom="@dimen/spacing_96" />
-
   <TextView
     android:id="@+id/noRecentPatientsTextView"
-    style="@style/Clinic.V2.TextAppearance.Subtitle1Center.Grey1"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginBottom="104dp"
-    android:layout_marginEnd="@dimen/spacing_8"
     android:layout_marginStart="@dimen/spacing_8"
+    android:layout_marginEnd="@dimen/spacing_8"
+    android:layout_marginBottom="104dp"
     android:background="@drawable/results_row_empty_state_background"
     android:padding="@dimen/spacing_16"
-    android:text="@string/patients_recentpatients_no_recent_patients" />
+    android:text="@string/patients_recentpatients_no_recent_patients"
+    android:textAppearance="?attr/textAppearanceSubtitle1"
+    android:textColor="?attr/colorHighEmphasis"
+    android:visibility="gone" />
```

Comment:
We don't want to show the error when the data is loading no? Since it will be displayed once the recent patients query is done I thought this will be fine. WDYT?

Reply:
Yeah, it's fine. Just wanted to clarify why this was added at part of this migration since it's not related.